crates/astria-core/src/protocol/account/v1alpha1/mod.rs

```diff
@@ -55,6 +55,18 @@ impl AssetBalance {
     }
 }
 
+impl PartialOrd for AssetBalance {
```

Comment:
I think this `PartialOrd` is not correct because it completely ignores the balance itself.

Worse, it's actually at odds with the `PartialEq` and `Eq` derivation, because:


```rust
let asset_one = AssetBalance {
    denom: "mydenom".parse().unwrap(),
    balance: 42,
};
let asset_two = AssetBalance {
    denom: "mydenom".parse().unwrap(),
    balance: 42,
};
assert_eq!(asset_one.cmp(&asset_two), Ordering::Equal);
assert!(asset_one.neq(&asset_two));
```

Reply:
Removed in 0af3bcf.